admin/app/templates/components/certification-details-competence.hbs

```diff
@@ -39,7 +39,7 @@
       {{/if}}
       <div class="card-deck">
         {{#each this.answers as |answer|}}
-          <CertificationDetailsAnswer @answer={{answer}} @onUpdateRate={{@onUpdateRate}} />
+          <Certification::CertificationDetailsAnswer @answer={{answer}} @onUpdateRate={{@onUpdateRate}} />
```

Comment:
**Question:** Quelle est cette sorcellerie?

Reply:
C'est [la façon de faire](https://guides.emberjs.com/release/components/introducing-components/#toc_nesting-components-in-folders) pour accéder à un sous-répertoire, en l'occurence `app/components/certification/certification-detail-answer`.

![image](https://user-images.githubusercontent.com/265963/99314344-41741f00-2861-11eb-9f01-26e347e92222.png)